### PR TITLE
Remind about ska3-perl in PR template and Issue template

### PR DESCRIPTION
**Pull request by jeanconn**
Remind about ska3-perl in PR template and Issue template .

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/ska3-flight.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ The latest release candidates will be installed in `/proj/sot/ska3/test` on HEAD
 conda create -n ska3-flight-{version}rc# --override-channels \
   -c https://icxc.cfa.harvard.edu/aspect/ska3-conda/flight \
   -c https://icxc.cfa.harvard.edu/aspect/ska3-conda/test \
-  ska3-flight=={version}rc#
+  ska3-flight=={version}rc# ska3-perl=={version}rc#
```
Comment (jeanconn) [inline .github/PULL_REQUEST_TEMPLATE/ska3-flight.md:19]: For this, I suppose there's a legit process question.  We're making a new ska3-perl package for every flight release - should we just install it and save the question about "is there a perl package update?".  I'm suggesting here that we might as well.

Comment (javierggt) [reply .github/PULL_REQUEST_TEMPLATE/ska3-flight.md:19]: We rarely release ska3-perl, so most of the time this line will not work.

Comment (jeanconn) [reply .github/PULL_REQUEST_TEMPLATE/ska3-flight.md:19]: Ah.  Sorry, I thought we were just always building this - so this should be fixed.  Will do.

**Comment by jeanconn** [comment]
And in my case, since I did install testing on GRETA and HEAD, starcheck did not fail - it just ran on an older-but-working version of ska3-perl.

**Comment by jeanconn** [comment]
I think I've addressed your comments @javierggt .

**Review by javierggt — APPROVED** [review]

**Pull request merged**